controls/radgridview/styles-and-templates/styling-row-details.md

```diff
@@ -42,7 +42,7 @@ You have two options:
 >If you're using [Implicit Styles]({%slug styling-apperance-implicit-styles-overview%}), you should base your style on the __DetailsPresenterStyle__.
 
 ## Setting a Column's RowDetailsStyle
-
+<Comment: Shouldn't the previous heading be Setting RadGridView's RowDetailsStyle? That's what the section seems to be about, and that is what is described in the list at the beginning of this article.>
```

Comment:
The previous section concerns creating the style while this one discusses how to set that style as the **RowDetailsStyle** property of a given column.